source/lib/ze_lib.cpp

```diff
@@ -401,6 +404,81 @@ zelSetDelayLoaderContextTeardown()
     #endif
 }
 
+#ifdef DYNAMIC_LOAD_LOADER
+void stabilityCheck(std::promise<int> stabilityPromise) {
+    try {
+        if (!ze_lib::context->loader) {
+            if (ze_lib::context->debugTraceEnabled) {
+                std::string message = "Loader module is null. Exiting stability checker thread.";
+                ze_lib::context->debug_trace_message(message, "");
+            }
+            stabilityPromise.set_value(1);
```

Comment:
Can we change the 1, 2, 3 values being set to #defines or enums to give descriptive name?

Reply:
added defines for the return codes.